### DISCUSSION TITLE
The ListPart in custom widget reported errors "The page could not be found".

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#15232] [General]
### Describe the bug
The ListPart in custom widget reported errors "The page could not be found".

### To Reproduce
1. Create a new content type with Widget stereotype and name ListWidget.
2. Add ListPart to ListWidget.
3. Set "Contained Content Types" of ListPart Settings to "Product".
4. Create a new content item of Page and add a ListWidget to Flow of the content item of Page.
![image](https://github.com/OrchardCMS/OrchardCore/assets/51013754/4b3f045a-b10f-4b20-b2da-8b09c47bc86c)

6. Click on 'List Items' then display ""The page could not be found".
![image](https://github.com/OrchardCMS/OrchardCore/assets/51013754/887a5132-e48c-416d-9da5-3228b02be580)

7. Click on 'Edit List Widget' then display ""The page could not be found".
8. Click on 'Create Product' to add a couple of product content items, but the widget don't display these content items.
![image](https://github.com/OrchardCMS/OrchardCore/assets/51013754/b4d0f359-7fde-4ed3-a83b-2c2dfe21ce16)

**Comment by MikeAlhayek** [comment]
I have no idea why you would add ListPart using Flow Part.

I am guessing you are doing something fundamentally wrong.

List part will allow you to create multiple contents under a single content. For example, if your content item is called Company and it has a list part with Location as a contained content type, then under each Company content item you'll have the ability to add/removed/edit multiple locations using the list part.

Maybe you can elaborate about your use case so we can provide you more guidance. This isn't a issue so I am converting this issue to a discussion.

**Reply by effapp** [reply]
Hi @MikeAlhayek ,
 
      The reason why I add ListPart using FlowPart to display a couple of list of content item in a certain parts of Page when I design Page in admin UI.
for example :
The articles page display serveral article list by category, like as shown in the following figure:
![image](https://github.com/OrchardCMS/OrchardCore/assets/51013754/118ffc71-cbd6-454f-b4cf-10268bececcb)

**Reply by MikeAlhayek** [reply]
If you are looking for a way to group your articles, then you should use taxonomy not list part

**Reply by effapp** [reply]
Hi @MikeAlhayek ,

I have grouped articles with taxonomy in article content type.I'll display article list with ListPart In front-end page that I design  in admin UI.
Therefore, I put ListPart in Custom Widget,but it reported "The page could not be found" error.

**Reply by MikeAlhayek** [reply]
ListPart is for Admin only. It should not be used for the front end. If you want to list content items on the front end, you may create a widget and place it on a zone on the front end. You can also use layers to control when you display it and who should see it.

You could create a sql query using the SQL query feature and display the result inside of a widget

Or you can create your own c# widget that renders items too

**Comment by effapp** [comment]
Thanks.
